plugins/inputs/clickhouse/clickhouse.go

```diff
@@ -635,18 +648,18 @@ const (
 	systemZookeeperRootNodesSQL = "SELECT count() AS zk_root_nodes FROM system.zookeeper WHERE path='/'"
 
 	systemReplicationExistsSQL   = "SELECT count() AS replication_queue_exists FROM system.tables WHERE database='system' AND name='replication_queue'"
-	systemReplicationNumTriesSQL = "SELECT countIf(num_tries>1) AS replication_num_tries_replicas, countIf(num_tries>100) AS replication_too_many_tries_replicas FROM system.replication_queue"
+	systemReplicationNumTriesSQL = "SELECT countIf(num_tries>1) AS replication_num_tries_replicas, countIf(num_tries>100) AS replication_too_many_tries_replicas FROM system.replication_queue SETTINGS empty_result_for_aggregation_by_empty_set=0"
```

Comment:
Is `SETTINGS empty_result_for_aggregation_by_empty_set=0` needed? From looking at the [docs](https://clickhouse.tech/docs/en/whats-new/changelog/2018/#new-features-16), setting it to 0 preserves the new behavior which seems to be on by default.

Reply:
this change fix the corner case when `replication_queue` is empty to SQL standards